French translation

spidercraft opened the pull request.

spidercraft committed:
French translation

Philanche reviewed (changes requested):
Looks very good overall, just a couple of tiny nitpicks.

spidercraft commented on src/main/resources/assets/voxy/lang/fr_fr.json:
```diff
@@ -0,0 +1,29 @@
+  "voxy.config.threads.saving": "Sauvegarde",
+  "voxy.config.threads.saving.tooltip": "Combien de threads voxy va utilisera pour sauvegarder des sections dans le stockage",
+  "voxy.config.threads.render": "Rendu",
+  "voxy.config.threads.render.tooltip": "Combien de threads voxy va utilisera pour générer des nouvelles données de rendu"
```
oops my bad, i did a couple rephrasing and forgot to remove the "va"

spidercraft committed:
Update fr_fr.json

spidercraft committed:
Update fr_fr.json

replaced "Tampon" by "Mémoire Tampon" wich is a better way to represent the buffer in this context

spidercraft committed:
Update fr_fr.json

Added caps for "Voxy" and "Nvidia"

ff66theone commented:
Pretty much good, however for the "mesh shader" options we should maybe add something that says it's only available for the GTX 1650 and up, much like it's said for the Nvidium mod (and add this to the original english translation as well).
